frontendGraph/src/main/java/com/virtuslab/gitmachete/frontend/graph/elements/BranchElement.java

```diff
@@ -17,17 +17,21 @@
   private final BaseGitMacheteBranch branch;
   private final SyncToOriginStatus syncToOriginStatus;
   private final SimpleTextAttributes attributes;
+  private final boolean hasSubelement;
 
   public BranchElement(
       BaseGitMacheteBranch branch,
       GraphEdgeColor graphEdgeColor,
       int upElementIndex,
       SyncToOriginStatus syncToOriginStatus,
-      boolean isCurrentBranch) {
-    super(graphEdgeColor, upElementIndex);
+      boolean isCurrentBranch,
+      int indentLevel,
+      boolean hasSubelement) {
```

Comment:
Hmm there's `downElement` naming out there but also `subelement` here... are these two intended to mean different things?

Reply:
yes, please see comments below https://github.com/VirtusLab/git-machete-intellij-plugin/pull/194#discussion_r409859928